_posts/2024-10-28-WildFly-mini-conference-save-the-date.adoc

```diff
@@ -0,0 +1,28 @@
+---
+layout: post
+title:  "Announcement: WildFly Mini Conference - Save the Date"
+date:   2024-10-28
+tags:   wildfly announcement conference wildfly-mini-conference
+author: frainone
+description: Save the Date: WildFly Mini Conference Nov 20th
+---
+
+= Save the Date: WildFly Mini Conference on Nov 20th
+
+Hello, WildFly Community!
+
+We are happy to announce that the next WildFly Mini Conference is scheduled for November 20th, 2024! Building on the success of our previous mini conference event, we are eager to bring together our community for another day of insightful discussions and online networking.
+
+As you know, a few weeks ago we sent a link:http://localhost:4000/news/2024/10/07/WildFly-mini-conference-session-form[request for feedback, window=_blank]. Based on the  feedback we received, the most voted topics were:
```

Comment:
@fl4via - this link is not correct as it points to localhost:4000 (https://www.wildfly.org/news/2024/10/28/2024-10-28-WildFly-mini-conference-save-the-date/) /cc @bstansberry

Reply:
I fixed that via #683. Thanks @tomjenkinson